### ISSUE TITLE
Array and notEmpty validator

### ISSUE THREAD

**Issue by aurmil** [Respect/Validation#306]
hi

I'm struggling with the following problem:

```php
$data = '';
var_dump(v::notEmpty()->validate($data)); // bool(false)

$data = ['test' => ''];
var_dump(v::arr()->key('test', v::notEmpty())->validate($data)); // bool(true)
```

Second validation test should be false too as "test" entry is empty...
Am I doing something wrong?
Using last package version through composer

thank you

**Comment by henriquemoody** [comment]
You're doing it right. Looks like this bug was introduced on commit da193b7

**Comment by henriquemoody** [comment]
This issue was fixed on `0.8.2` and `0.7.6` versions, please upgrade your Respect\Validation
